frontend/console/src/features/timeline/TimelineCronScheduled.tsx

```diff
@@ -0,0 +1,17 @@
+import type { CronScheduledEvent } from '../../protos/xyz/block/ftl/v1/console/console_pb'
+import { formatTimestampShort } from '../../utils/date.utils.ts'
+import { verbRefString } from '../verbs/verb.utils'
+
+export const TimelineCronScheduled = ({ cron }: { cron: CronScheduledEvent }) => {
+  const title = `Cron (${cron.schedule}) scheduled at ${cron.scheduledAt}`
+  return (
+    <span title={title}>
```

Comment:
uber-nit: we can do 
```<span title={`Cron (${cron.schedule}) scheduled at ${cron.scheduledAt}`}>``` since it's short anyway and we aren't doing any funky string manipulation that would be worth pulling into a separate line